### PR TITLE
polyhedral geometry: added constructions delpezzo and fano_simplex

### PR DESCRIPTION
**Pull request by micjoswig**
@lkastner Please check.  If you like it, please merge.  If you already did it, you can also kill this.

### CODE REVIEW

File: src/PolyhedralGeometry/Polyhedron/standard_constructions.jl
Comment (fingolfin) [inline src/PolyhedralGeometry/Polyhedron/standard_constructions.jl]:
```suggestion
```

File: src/PolyhedralGeometry/Polyhedron/standard_constructions.jl
Comment (fingolfin) [inline src/PolyhedralGeometry/Polyhedron/standard_constructions.jl]:
```suggestion
```

**Review by fingolfin — APPROVED** [review]
Looks good to me. I'll happily let @lkastner have the final word, though!

The extra newlines are unimportant and appear in other docstrings in that file and can be removed everywhere in a future PR.

**Review by lkastner — APPROVED** [review]
I will merge once the tests have finished running.

**Pull request merged**
